### PR TITLE
Adds more anti-toxin to green first aid kit

### PR DESCRIPTION
**Pull request by yorii**
## About The Pull Request

This PR replaces 1 bottle of charcoal pills with 1 additional syringe of anti-toxin

## Why It's Good For The Game

Charcoal can now be found plenty elsewhere and is not as effective at treating toxin damage as it used to be.

## Changelog
:cl:
tweak: tweaked the contents of toxin firstaid kits
/:cl:

### CODE REVIEW

**Comment by Archanial** [comment]
First aid is first aid, it covers basic damage - brute and burn. Toxin damage should be harder to cure. This makes it the easiest to cure.

@Rukofamicom is doing chemical rebalancing, what's your opinion on this?

**Comment by Effemu** [comment]
> 
> 
> First aid is first aid, it covers basic damage - brute and burn. Toxin damage should be harder to cure. This makes it the easiest to cure.
> 
> @Rukofamicom is doing chemical rebalancing, what's your opinion on this?

I think he means toxin treatment kit, not first aid.
Right now it contains 1 or 2 (cant remember) antitox syringes, 1 syringe of diethylamine, 1 syringe of calomel and 2 charcoal pill bottles plus health analyzer.

**Comment by PowerfulBacon** [comment]
wouldnt all the liquid come out if its in a glass bottle in a medkit?

**Comment by Sinestia** [comment]
This might be pedantic but it should be pills or patches instead of bottles to fit the first aid theme and make it more disposable and less optimized. 


Also your why it's good for the game thing doesn't really explain why toxin kits need to be buffed.

 The charcoal nerf came in the same change that messed with the toxin kit so this is all intentional, not really an oversight.

**Comment by ilikesaying** [comment]
Adds more anti-toxin to green first aid **kid**

File: code/game/objects/items/storage/firstaid.dm [hunk 1]
```diff
@@ -172,10 +172,10 @@
 	if(empty)
 		return
 	var/static/items_inside = list(
-		/obj/item/reagent_containers/syringe/antitoxin = 2,
+		/obj/item/reagent_containers/glass/bottle/antitoxin = 3,
```
Comment (Crossedfall) [inline code/game/objects/items/storage/firstaid.dm]: Stick with syringes or pills for medkits

**Review by Crossedfall — DISMISSED** [review]

File: code/game/objects/items/storage/firstaid.dm (hunk 1 above)
Comment (yorii) [reply code/game/objects/items/storage/firstaid.dm]: so... maybe just more syringes and less charcoal then?

**Comment by yorii** [comment]
@TheFakeElon you can remove Stale, I changed it to just replace 1 charcoal bottle with 1 antitoxin syringe now.

**Review by utilbird — APPROVED** [review]

**Comment by utilbird** [comment]
^Code approval btw. I agree with archanial on this "balance wise'

**Review by utilbird — APPROVED** [review]

**Pull request merged**